Source/Csla.Analyzers/Csla.Analyzers/AsynchronousBusinessRuleInheritingFromBusinessRuleAnalyzer.cs

```diff
@@ -7,6 +7,9 @@
 
 namespace Csla.Analyzers
 {
+  /// <summary>
```

Comment:
Why aren't you just disabling the doc generation for this particular project? Empty comments have no value at all?

Reply:
Would that have disabled the compiler warnings?